test/core/xds/xds_channel_stack_modifier_test.cc

```diff
@@ -111,44 +110,6 @@ TEST(XdsChannelStackModifierTest, XdsHttpFiltersInsertion) {
   grpc_shutdown();
 }
 
-// Test filters insertion with OpenCensus plugin registered
```

Comment:
Shouldn't we still test this with "census_server", since the code is still checking for that case?

Reply:
This test just checks for OpenCensus unfortunately :( For the internal census, we will need a test in google3.

Reply:
cl/516905576 moves it to an internal test